_episodes_rmd/10-supp-addressing-data.Rmd

```diff
@@ -45,13 +45,15 @@ Lets take a look at this data.
 class(dat)
 ```
 
-R has loaded the contents of the .csv file into a variable called `dat` which is a `data frame`.
+R has loaded the contents of the .csv file into a variable called `dat` which is a `data frame`. A data frame is a list of variables of the same number of rows with unique row names, given class "data.frame".
```

Comment:
Is a `the` or similar missing in the end?

Reply:
Also, let's try to not repeat "data frame" or "data.frame".

Reply:
The sentence " A data frame is a list of variables of the same number of rows with unique row names, given class "data.frame" was taken from the R help page on data frames (https://stat.ethz.ch/R-manual/R-devel/library/base/html/data.frame.html). I am happy to remove it from the pull request if it is not helpful or unclear.

Reply:
Yes, in that case, please remove it. We don't need to duplicate content from the official manual. On the contrary, it seems to require some [improvement suggestions](https://www.r-project.org/bugs.html) itself ;-)